week-1/L-functions-nested/exercise.js

```diff
@@ -0,0 +1,21 @@
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+
+function Students() {
```

Comment:
Function names in JavaScript should start with a small letter. So it should be `function students() {` instead of `function Students() {`